src/platform_impl/windows/event_loop.rs

```diff
@@ -1640,68 +1648,23 @@ unsafe fn public_window_callback_inner<T: 'static>(
 
                 SkipPointerFrameMessages(pointer_id);
             }
-            0
+            result = ProcResult::Value(0);
         }
 
         winuser::WM_SETFOCUS => {
-            use crate::event::{ElementState::Released, WindowEvent::Focused};
-            for windows_keycode in event::get_pressed_keys() {
-                let scancode =
-                    winuser::MapVirtualKeyA(windows_keycode as _, winuser::MAPVK_VK_TO_VSC);
-                let virtual_keycode = event::vkey_to_winit_vkey(windows_keycode);
-
-                update_modifiers(window, userdata);
-
-                #[allow(deprecated)]
-                userdata.send_event(Event::WindowEvent {
-                    window_id: RootWindowId(WindowId(window)),
-                    event: WindowEvent::KeyboardInput {
-                        device_id: DEVICE_ID,
-                        input: KeyboardInput {
-                            scancode,
-                            virtual_keycode,
-                            state: Released,
-                            modifiers: event::get_key_mods(),
-                        },
-                        is_synthetic: true,
-                    },
-                })
-            }
```

Comment:
I was a bit hesitant with this. Why did we send a RELEASED event for the pressed keys during WM_SETFOCUS?

Reply:
As I understand it, it's to notify the user that they ought to treat the keys as released when focus is lost.

Reply:
But this happened on SETFOCUS not just KILLFOCUS.

Reply:
Welp, that's strange. I'd say we don't do anything until someone complains.